Use one queue per messaging backend

jeremycline opened the pull request:
Publish messages to the backend using a routing key based off the
backend (``fmn.backend.`` + ``<backend-name>``). This solves the issue
introduced in #238 where the irc network being down causes emails to
back up. The delivery service round-robins the various queues.

cc @puiterwijk  @pypingou

Signed-off-by: Jeremy Cline <jeremy@jcline.org>

jeremycline committed:
Use one queue per messaging backend

Publish messages to the backend using a routing key based off the
backend (``fmn.backend.`` + ``<backend-name>``). This solves the issue
introduced in #238 where the irc network being down causes emails to
back up. The delivery service round-robins the various queues.

Signed-off-by: Jeremy Cline <jeremy@jcline.org>

bowlofeggs reviewed (approved):
Looks fine to me, though the CI tests seem to fail so I recommend sorting those out.

jeremycline committed:
Ignore style in fmn/consumer.py until it is PEP8

For some reason fmn/consumer.py was previously not getting linted. This
adds it to the ignore list until it can be fixed.

Signed-off-by: Jeremy Cline <jeremy@jcline.org>
